core/src/banking_stage/consumer.rs

```diff
@@ -0,0 +1,2074 @@
+use {
+    super::{
+        committer::{CommitTransactionDetails, Committer},
+        BankingStageStats,
```

Comment:
a remaining thing that *may* be nice to pull out of the top-level file...if for nothing else, to remove this circular dependencies.